Library/Homebrew/cmd/audit.rb

```diff
@@ -121,10 +121,15 @@ def include?(s)
     @text.include? s
   end
 
-  def line_number(regex)
-    index = @lines.index { |line| line =~ regex }
+  def line_number(regex, skip = 0)
+    index = @lines.drop(skip).index { |line| line =~ regex }
```

Comment:
The following will avoid constructing an intermediate array with `drop`:

```ruby
index = @lines.each_with_index.find_index { |line, index| index >= skip && line =~ regex }
```

(But if you prefer to stick with your version, it has a bug as `skip` needs to be added to the returned index.)